docs/maintainers/ci.md

```diff
@@ -0,0 +1,106 @@
+# Continuous Integration
```

Comment:
This file needs to be renamed to `continuous-integration.md`

Reply:
Done